src/dagent/docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+# docker-compose.yml
+version: '3.3'
+services:
+  dagent:
+    restart: always
+    #image: tiangolo/uwsgi-nginx-flask:python3.7
+    #image: golang:latest
+    image: m9k-golang
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./:/app1
+      - /tmp/air_result.json:/tmp/air_result.json
```

Comment:
Rather than using the host's tmp or etc directory, I think it would be better to make tmp, etc. where the 'docker-compose' file is located and link it there.

**for example**
./tmp/air_result.json (Host) to /tmp/air_result.json (Container)
./etc/ibofos (Host) to /etc/ibofos (Container)

Reply:
Isn't /etc/pos right now? Does /etc/ibofos still exist?

Reply:
It is assumed that POS runs on local baremetal server, so host's tmp or etc can be linked.
/etc/ibofos -> /etc/pos confirmed